### PR TITLE
Add ability to ignore certificates during resource check using property

### PR DESCRIPTION
**Pull request by AlenaBudevich**

### CODE REVIEW

File: vividus-plugin-web-app-to-rest-api/src/main/resources/spring.xml [hunk 1]
```diff
@@ -63,6 +63,7 @@
                 <property name="skipResponseEntity" value="true" />
                 <property name="lastRequestInterceptor"><null/></property>
                 <property name="lastResponseInterceptor"><null/></property>
+                <property name="sslCertificateCheckEnabled" value="${http.resource-validator.ssl-certificate-check}" />
```
Comment (ikalinin1) [inline vividus-plugin-web-app-to-rest-api/src/main/resources/spring.xml:66]: `http.ssl.check-certificate` is not working?

Comment (AlenaBudevich) [reply vividus-plugin-web-app-to-rest-api/src/main/resources/spring.xml:66]: This property doesn't help :(

File: vividus-plugin-web-app-to-rest-api/src/main/resources/properties/defaults/run.properties [hunk 2]
```diff
@@ -8,3 +8,5 @@ bdd.transformer.headless.filter-redirects=false
 bdd.transformer.headless.seed-relative-urls=
 
 resource-checker.uri-to-ignore-regex=
+
+http.resource-validator.ssl-certificate-check=true
```
Comment (valfirst) [inline vividus-plugin-web-app-to-rest-api/src/main/resources/properties/defaults/run.properties:12]: `resource-checker.http.ssl.check-certificate`

File: vividus-plugin-web-app-to-rest-api/src/main/resources/spring.xml (hunk 1 above)
Comment (valfirst) [reply vividus-plugin-web-app-to-rest-api/src/main/resources/spring.xml:66]: That's weird

**Pull request closed without merging**